src/main/java/io/rcktapp/api/handler/sql/SqlDb.java

```diff
@@ -305,76 +310,123 @@ public void reflectDb() throws Exception
          //-- that caputres all of the foreign key relationships.  You
          //-- have to do the fk loop second becuase the reference pk
          //-- object needs to exist so that it can be set on the fk Col
-         ResultSet rs = dbmd.getTables(null, "public", "%", new String[]{"TABLE", "VIEW"});
-         while (rs.next())
-         {
-            String tableCat = rs.getString("TABLE_CAT");
-            String tableSchem = rs.getString("TABLE_SCHEM");
-            String tableName = rs.getString("TABLE_NAME");
-            //String tableType = rs.getString("TABLE_TYPE");
-
-            Table table = new Table(this, tableName);
-            addTable(table);
-
-            ResultSet colsRs = dbmd.getColumns(tableCat, tableSchem, tableName, "%");
-
-            while (colsRs.next())
-            {
-               String colName = colsRs.getString("COLUMN_NAME");
-               Object type = colsRs.getString("DATA_TYPE");
-               String colType = types.get(type);
-
-               boolean nullable = colsRs.getInt("NULLABLE") == DatabaseMetaData.columnNullable;
-
-               Column column = new Column(table, colName, colType, nullable);
-               table.addColumn(column);
-
-               //               if (DELETED_FLAGS.contains(colName.toLowerCase()))
-               //               {
-               //                  table.setDeletedFlag(column);
-               //               }
+            Map<String, CompletableFuture<Table>> tableFutures = new HashMap<>();
+            try (ResultSet rs = dbmd.getTables(null, "public", "%", new String[]{"TABLE", "VIEW"})) {
+                while (rs.next())
+                {
+                    String tableCat = rs.getString("TABLE_CAT");
+                    String tableSchem = rs.getString("TABLE_SCHEM");
+                    String tableName = rs.getString("TABLE_NAME");
+                    //String tableType = rs.getString("TABLE_TYPE");
+
+                    if (Stream.of(ignoreTablePrefixes.split(",")).filter(Predicate.not(String::isEmpty)).anyMatch(tableName::startsWith)) continue;
```

Comment:
Added configuration element 'ignoreTablePrefixes' and configured it to ignore PRICEBOOK_INDEXER_ prefixed tables.  That way we don't waste our time (previously ~20 seconds) loading schema of tables snooze never references.